modules/Store.js

```diff
@@ -0,0 +1,26 @@
+export default class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(book) {
+    const bookTitle = book.querySelector('#title').innerText;
+    const books = Store.getBooks();
+    const filt = books.filter((book) => bookTitle === book.title);
+    const filtIndex = books.indexOf(filt[0]);
+    books.splice(filtIndex, 1);
+    localStorage.setItem('books', JSON.stringify(books));
```

Comment:
- [ ] Kindly check the method for removing the book because if I have 3 books and try to remove book number 2 and then refresh the page I find the last book removed in the second one, you can open local storage from the browser and check it.

Reply:
> * [ ]  Kindly check the method for removing the book because if I have 3 books and try to remove book number 2 and then refresh the page I find the last book removed in the second one, you can open local storage from the browser and check it.

 
could you please help me  what to do